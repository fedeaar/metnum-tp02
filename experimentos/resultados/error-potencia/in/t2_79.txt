7.761630000000000000e+06 -5.308320000000000000e+05 1.055880000000000000e+05 6.185620000000000000e+05 -9.237700000000000000e+04 -3.746480000000000000e+05 2.216019000000000000e+06 2.037410000000000000e+05 2.923000000000000000e+05 -5.085920000000000000e+05 4.575600000000000000e+04 -1.626889000000000000e+06 -1.677793000000000000e+06 1.182544000000000000e+06 -3.170900000000000000e+05 1.719221000000000000e+06 -3.014200000000000000e+05 -2.032928000000000000e+06 1.502062000000000000e+06 1.302013000000000000e+06
-5.308320000000000000e+05 5.256097000000000000e+06 1.028688000000000000e+06 1.760710000000000000e+06 -5.367910000000000000e+05 -3.304892000000000000e+06 1.094746000000000000e+06 4.751620000000000000e+05 -1.097827000000000000e+06 -3.705165000000000000e+06 1.107954000000000000e+06 5.297090000000000000e+05 4.423850000000000000e+05 1.058831000000000000e+06 1.866892000000000000e+06 -1.403930000000000000e+06 4.103690000000000000e+05 1.616977000000000000e+06 1.036106000000000000e+06 1.296130000000000000e+05
1.055880000000000000e+05 1.028688000000000000e+06 7.668150000000000000e+06 1.543671000000000000e+06 1.470393000000000000e+06 -2.649786000000000000e+06 -3.329680000000000000e+05 3.789322000000000000e+06 -1.062330000000000000e+06 -1.767926000000000000e+06 -1.341337000000000000e+06 1.794155000000000000e+06 5.052010000000000000e+05 -8.683500000000000000e+05 -6.504000000000000000e+05 -1.684540000000000000e+05 8.348100000000000000e+04 -1.035551000000000000e+06 -4.619100000000000000e+05 -1.658612000000000000e+06
6.185620000000000000e+05 1.760710000000000000e+06 1.543671000000000000e+06 6.872849000000000000e+06 -3.075050000000000000e+05 -2.027845000000000000e+06 6.579760000000000000e+05 1.260481000000000000e+06 -1.589131000000000000e+06 -2.629648000000000000e+06 1.458221000000000000e+06 1.372131000000000000e+06 -2.447080000000000000e+06 -2.621587000000000000e+06 3.070165000000000000e+06 6.790080000000000000e+05 1.247395000000000000e+06 2.705823000000000000e+06 -7.760990000000000000e+05 -1.155885000000000000e+06
-9.237700000000000000e+04 -5.367910000000000000e+05 1.470393000000000000e+06 -3.075050000000000000e+05 5.347702000000000000e+06 1.374410000000000000e+05 -1.365659000000000000e+06 1.339429000000000000e+06 3.788520000000000000e+05 -6.313060000000000000e+05 -1.284200000000000000e+06 1.404660000000000000e+06 -1.633190000000000000e+05 1.139928000000000000e+06 -2.071580000000000000e+05 1.888899000000000000e+06 3.415300000000000000e+04 -7.069600000000000000e+05 2.551870000000000000e+05 -1.195759000000000000e+06
-3.746480000000000000e+05 -3.304892000000000000e+06 -2.649786000000000000e+06 -2.027845000000000000e+06 1.374410000000000000e+05 7.683645000000000000e+06 -2.706350000000000000e+05 -3.151421000000000000e+06 2.224029000000000000e+06 8.352160000000000000e+05 4.188440000000000000e+05 -1.388584000000000000e+06 1.659687000000000000e+06 8.327790000000000000e+05 -6.101930000000000000e+05 -1.019681000000000000e+06 -2.320302000000000000e+06 -9.741960000000000000e+05 1.793790000000000000e+05 -1.252512000000000000e+06
2.216019000000000000e+06 1.094746000000000000e+06 -3.329680000000000000e+05 6.579760000000000000e+05 -1.365659000000000000e+06 -2.706350000000000000e+05 8.925371000000000000e+06 2.963119000000000000e+06 -1.120986000000000000e+06 1.296911000000000000e+06 1.758273000000000000e+06 -1.614962000000000000e+06 1.491898000000000000e+06 1.093624000000000000e+06 9.491640000000000000e+05 -5.101370000000000000e+05 2.329414000000000000e+06 -1.270726000000000000e+06 1.508053000000000000e+06 -7.923290000000000000e+05
2.037410000000000000e+05 4.751620000000000000e+05 3.789322000000000000e+06 1.260481000000000000e+06 1.339429000000000000e+06 -3.151421000000000000e+06 2.963119000000000000e+06 6.278024000000000000e+06 -3.162290000000000000e+05 2.032280000000000000e+05 6.537460000000000000e+05 -8.438800000000000000e+04 8.208330000000000000e+05 -5.138610000000000000e+05 -1.757460000000000000e+05 -2.172900000000000000e+05 3.181477000000000000e+06 -1.126125000000000000e+06 8.830690000000000000e+05 -1.067979000000000000e+06
2.923000000000000000e+05 -1.097827000000000000e+06 -1.062330000000000000e+06 -1.589131000000000000e+06 3.788520000000000000e+05 2.224029000000000000e+06 -1.120986000000000000e+06 -3.162290000000000000e+05 5.545991000000000000e+06 4.612880000000000000e+05 -4.628830000000000000e+05 1.356061000000000000e+06 5.316770000000000000e+05 4.040240000000000000e+05 4.810410000000000000e+05 -7.712090000000000000e+05 1.685107000000000000e+06 1.002290000000000000e+06 2.155952000000000000e+06 2.266050000000000000e+06
-5.085920000000000000e+05 -3.705165000000000000e+06 -1.767926000000000000e+06 -2.629648000000000000e+06 -6.313060000000000000e+05 8.352160000000000000e+05 1.296911000000000000e+06 2.032280000000000000e+05 4.612880000000000000e+05 8.197820000000000000e+06 -7.775390000000000000e+05 -1.717773000000000000e+06 -4.443810000000000000e+05 -1.394000000000000000e+06 -6.962030000000000000e+05 -8.403200000000000000e+05 -8.998500000000000000e+04 1.074190000000000000e+05 -1.955716000000000000e+06 1.950337000000000000e+06
4.575600000000000000e+04 1.107954000000000000e+06 -1.341337000000000000e+06 1.458221000000000000e+06 -1.284200000000000000e+06 4.188440000000000000e+05 1.758273000000000000e+06 6.537460000000000000e+05 -4.628830000000000000e+05 -7.775390000000000000e+05 7.916434000000000000e+06 2.718400000000000000e+04 1.636274000000000000e+06 -6.898000000000000000e+05 3.199800000000000000e+04 -3.768010000000000000e+05 1.935222000000000000e+06 1.956173000000000000e+06 -1.127801000000000000e+06 2.708600000000000000e+05
-1.626889000000000000e+06 5.297090000000000000e+05 1.794155000000000000e+06 1.372131000000000000e+06 1.404660000000000000e+06 -1.388584000000000000e+06 -1.614962000000000000e+06 -8.438800000000000000e+04 1.356061000000000000e+06 -1.717773000000000000e+06 2.718400000000000000e+04 8.800476000000000000e+06 1.595604000000000000e+06 -1.873576000000000000e+06 -8.120980000000000000e+05 2.852900000000000000e+05 1.477961000000000000e+06 1.405406000000000000e+06 -1.410100000000000000e+04 -3.043550000000000000e+05
-1.677793000000000000e+06 4.423850000000000000e+05 5.052010000000000000e+05 -2.447080000000000000e+06 -1.633190000000000000e+05 1.659687000000000000e+06 1.491898000000000000e+06 8.208330000000000000e+05 5.316770000000000000e+05 -4.443810000000000000e+05 1.636274000000000000e+06 1.595604000000000000e+06 5.601700000000000000e+06 1.564723000000000000e+06 -2.108782000000000000e+06 -1.559389000000000000e+06 6.448980000000000000e+05 -9.663750000000000000e+05 2.244390000000000000e+06 -4.868340000000000000e+05
1.182544000000000000e+06 1.058831000000000000e+06 -8.683500000000000000e+05 -2.621587000000000000e+06 1.139928000000000000e+06 8.327790000000000000e+05 1.093624000000000000e+06 -5.138610000000000000e+05 4.040240000000000000e+05 -1.394000000000000000e+06 -6.898000000000000000e+05 -1.873576000000000000e+06 1.564723000000000000e+06 3.968191000000000000e+06 -1.109292000000000000e+06 6.189790000000000000e+05 -8.842830000000000000e+05 -2.505602000000000000e+06 2.097944000000000000e+06 4.859450000000000000e+05
-3.170900000000000000e+05 1.866892000000000000e+06 -6.504000000000000000e+05 3.070165000000000000e+06 -2.071580000000000000e+05 -6.101930000000000000e+05 9.491640000000000000e+05 -1.757460000000000000e+05 4.810410000000000000e+05 -6.962030000000000000e+05 3.199800000000000000e+04 -8.120980000000000000e+05 -2.108782000000000000e+06 -1.109292000000000000e+06 4.967851000000000000e+06 -8.314600000000000000e+04 6.493650000000000000e+05 2.439236000000000000e+06 3.539110000000000000e+05 -1.035251000000000000e+06
1.719221000000000000e+06 -1.403930000000000000e+06 -1.684540000000000000e+05 6.790080000000000000e+05 1.888899000000000000e+06 -1.019681000000000000e+06 -5.101370000000000000e+05 -2.172900000000000000e+05 -7.712090000000000000e+05 -8.403200000000000000e+05 -3.768010000000000000e+05 2.852900000000000000e+05 -1.559389000000000000e+06 6.189790000000000000e+05 -8.314600000000000000e+04 8.204317000000000000e+06 7.784200000000000000e+05 -1.850787000000000000e+06 -1.425110000000000000e+06 -1.237603000000000000e+06
-3.014200000000000000e+05 4.103690000000000000e+05 8.348100000000000000e+04 1.247395000000000000e+06 3.415300000000000000e+04 -2.320302000000000000e+06 2.329414000000000000e+06 3.181477000000000000e+06 1.685107000000000000e+06 -8.998500000000000000e+04 1.935222000000000000e+06 1.477961000000000000e+06 6.448980000000000000e+05 -8.842830000000000000e+05 6.493650000000000000e+05 7.784200000000000000e+05 6.209408000000000000e+06 2.875160000000000000e+06 1.924363000000000000e+06 2.169351000000000000e+06
-2.032928000000000000e+06 1.616977000000000000e+06 -1.035551000000000000e+06 2.705823000000000000e+06 -7.069600000000000000e+05 -9.741960000000000000e+05 -1.270726000000000000e+06 -1.126125000000000000e+06 1.002290000000000000e+06 1.074190000000000000e+05 1.956173000000000000e+06 1.405406000000000000e+06 -9.663750000000000000e+05 -2.505602000000000000e+06 2.439236000000000000e+06 -1.850787000000000000e+06 2.875160000000000000e+06 9.139732000000000000e+06 6.369400000000000000e+05 2.355265000000000000e+06
1.502062000000000000e+06 1.036106000000000000e+06 -4.619100000000000000e+05 -7.760990000000000000e+05 2.551870000000000000e+05 1.793790000000000000e+05 1.508053000000000000e+06 8.830690000000000000e+05 2.155952000000000000e+06 -1.955716000000000000e+06 -1.127801000000000000e+06 -1.410100000000000000e+04 2.244390000000000000e+06 2.097944000000000000e+06 3.539110000000000000e+05 -1.425110000000000000e+06 1.924363000000000000e+06 6.369400000000000000e+05 6.667998000000000000e+06 1.001879000000000000e+06
1.302013000000000000e+06 1.296130000000000000e+05 -1.658612000000000000e+06 -1.155885000000000000e+06 -1.195759000000000000e+06 -1.252512000000000000e+06 -7.923290000000000000e+05 -1.067979000000000000e+06 2.266050000000000000e+06 1.950337000000000000e+06 2.708600000000000000e+05 -3.043550000000000000e+05 -4.868340000000000000e+05 4.859450000000000000e+05 -1.035251000000000000e+06 -1.237603000000000000e+06 2.169351000000000000e+06 2.355265000000000000e+06 1.001879000000000000e+06 7.116521000000000000e+06
