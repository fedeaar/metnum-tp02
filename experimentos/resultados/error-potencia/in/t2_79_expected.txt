2.110076225524086505e+07
1.720163814297507331e+07
1.598014187783861160e+07
1.393558200026218034e+07
1.192522892974288575e+07
1.160122862727524899e+07
9.710095214227564633e+06
8.134157223882738501e+06
6.657996303848177195e+06
5.622421256483261473e+06
3.869497307625507936e+06
3.611071468107297551e+06
2.825270026114692446e+06
2.378419508539587259e+06
1.723101583045272622e+06
8.191795776511096628e+05
6.637755521269409219e+05
2.531194126648961974e+05
1.160353260805039026e+05
1.185406267785631371e+03
